specification/containerservices/resource-manager/Microsoft.ContainerService/stable/2018-03-31/managedClusters.json

```diff
@@ -750,28 +750,7 @@
         "Standard_NV6"
       ]
     },
-    "KeyVaultSecretRef": {
-      "properties": {
```

Comment:
A. Please note that this could be a breaking change in the SDKs generated
B. Please check if you need to update any of the examples

Reply:
@dsgouda I did check the examples, but we weren't using `keyVaultSecretRef` anywhere so this change still passes `oav validate-example`.

I did note that this is a breaking change in the PR message. Should I mention it elsewhere? Change to `package-version: 5.0.0`?

Reply:
As long as it is ackowledged, shouldn't be a blocker